docs/user_guide/tensorflow/model_serving.rst

```diff
@@ -24,73 +22,119 @@ To demonstrate this we provide an example notebook which is also included in the
 https://github.com/hopshadoop/hops-examples/blob/master/tensorflow/notebooks/Serving/train_and_export_model.ipynb
 
 In order to serve a TensorFlow model on Hopsworks, the .pb file and the variables folder should be placed in the Models dataset in your Hopsworks project. Inside the dataset, the folder structure should mirror what is expected by TensorFlow Serving.
-   
-.. _serving_structure.png: ../../_images/serving_structure.png
-.. figure:: ../../imgs/serving_structure.png
-   :alt: Expected file and folder structure for TensorFlow Serving
-   :target: `serving_structure.png`_
-   :align: center
-   :figclass: align-center
 
+.. code-block:: shell 
+
+    Models
+    └── mnist
+        ├── 1
+        │   ├── saved_model.pb
+        │   └── variables
+        │       ├── variables.data-00000-of-00001
+        │       └── variables.index
+        └── 2
+            ├── saved_model.pb
+            └── variables
+                ├── variables.data-00000-of-00001
+                └── variables.index
+
+TensorFlow serving expects the model directory (In this case *mnist*) to contain one or more sub-directories.
+The name of each sub-directory is a number representing the version of the model, the higher the version, the more recent the model.
+Inside each version directory TensorFlow serving expects a file named *saved_mode.pb*, which contains the graph of the model, and a directory called *variables* which contains the weights of the model.
```

Comment:
@SirOibaf instead of `graph of the model`, would `model graph` be more appropriate?